src/transformers/models/modernbert/modular_modernbert.py

```diff
@@ -799,10 +756,11 @@ def forward(
         """
         attn_outputs = self.attn(
             self.attn_norm(hidden_states),
+            attention_mask=attention_mask,
+            local_attention_mask=local_attention_mask,
```

Comment:
can we just switch to `sliding_window_mask` 😉

Reply:
Renamed in d799d65bd9ebeab85e1d6a7f453841c5aa21b272